Overview.bs

```diff
@@ -516,18 +516,21 @@ as follows:
 1.  If {{scrollSource}} is null, does not currently have a [=CSS layout box=], or
     if its layout box is not a [=scroll container=], return an unresolved time value.
 
-2.  Otherwise, let <var>current scroll offset</var> be the current scroll offset of {{scrollSource}}
+1.  Otherwise, let <var>current scroll offset</var> be the current scroll offset of {{scrollSource}}
     in the direction specified by {{orientation}}.
 
-3.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return an unresolved
-    time value if {{fill}} is <code>none</code> or <code>forwards</code>, 
-    or 0 otherwise.
+1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return an unresolved
+    time value if {{fill}} is <code>none</code> or <code>forwards</code>, or 0 otherwise.
 
-4.  If <var>current scroll offset</var> is greater than or equal to {{endScrollOffset}}, 
-    return an unresolved time value if {{fill}} is <code>none</code> or
+1.  If <var>current scroll offset</var> is greater than or equal to {{endScrollOffset}},
+    return an unresolved time value if {{endScrollOffset}} is less than the maximum scroll offset
+    of {{scrollSource}} in {{orientation}} and {{fill}} is <code>none</code> or
     <code>backwards</code>, or the [=effective time range=] otherwise.
```

Comment:
This is a pretty long sentence. It might be easier to follow if you broke it up such as:

1.  If `current scroll offset` is greater than or equal to `endScrollOffset` then:
    * If `endScrollOffset` is less than the maximum scroll offset of `scrollSource`, and `fill` is 'none' or 'backwards',
         * Return an unresolved time value.
    * Otherwise,
         * Return the `effective time range`.